docs/content/intro/why.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Why Does It Matter?"
+date: 2018-11-06T13:58:58-07:00
+weight: 4
+LastModifierDisplayName: "Marwan"
+LastModifierEmail: "marwan.sameer@gmail.com"
+
+---
+
+### Immutability
+
+Previously, the Go community has had lots of problems with libraries disappearing or changing without warning. It's easy for package maintainers to make changes to their code that can break yours - and much of the time it's an accident! Could your build break if one of your dependencies did this?
+
+- Commit `abdef` was deleted
```

Comment:
Let's not have this now as we do not support it?

Reply:
We can't fetch by commit hashes right? I am a bit confused?

Reply:
@manugupt1 we can if they are pseudo versions. This is a bit unrelated.